Sources/Rendering/WebGPU/CellArrayMapper/index.js

```diff
@@ -616,11 +617,14 @@ function vtkWebGPUCellArrayMapper(publicAPI, model) {
 
   // we only apply lighting when there is a "var normal" declaration in the
   // fragment shader code. That is the lighting trigger.
+  // TODO: Fix frame tanking when zooming in close to models
```

Comment:
Is this still a TODO? I am getting solid 60fps on my laptop regardless of zoom but I have a fairly good GPU

Reply:
Running it now on my main computer (which actually has a graphics unit) it runs flawlessly, so it was probably the laptop